app/javascript/App/components/CalendarPage/index.js

```diff
@@ -143,4 +145,6 @@ class CalendarPage extends Component {
   }
 }
 
-export default calendarContainer(CalendarPage)
+const CalendarPageWithStore = withStore(calendarContainer(CalendarPage))
+
+ReactOnRails.register({ CalendarPage: CalendarPageWithStore })
```

Comment:
This component uses redux. So, I'm wrapping it into `withStore` HOC to make it works until it will be refactored into `non-redux` version.

Also, the exporting of `CalendarPage` is not necessary because this component is used only by `react_component` function.

Reply:
I think it is still better to export just to maintain consistency with all the other components